components/Message/Message.styles.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components'
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+
+export const StyledList = styled(List)``
```

Comment:
Unless this is the style you prefer, this seems to add more of a trail to follow to find these styling components. It would be easier to follow if you just directly used the material-ui components in the component itself rather than renaming them.